tests/garage/profile.spec.js

```diff
@@ -0,0 +1,19 @@
+import {test, expect} from "../../src/fixtures/userGaragePage";
+import {PROFILE_MOCK_RESPONSE} from "./fixtures/profile";
+
+
+test.describe.only('Profile', ()=>{
+    test('open profile tab', async ({page})=>{
+
+        await page.route('https://qauto.forstudy.space/api/users/profile', (route)=>{
+            return route.fulfill({
+                status: 200,
+                body: JSON.stringify(PROFILE_MOCK_RESPONSE)
+            })
+        })
+        await page.goto('/panel/profile')
+
+        const profileName = await page.$eval('.profile_name', (element) => element.textContent.trim());
```

Comment:
use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text

and use POM